modules/drivers/presto/src/metabase/driver/presto.clj

```diff
@@ -267,6 +267,15 @@
   [_]
   :monday)
 
+(defmethod sql.qp/cast-temporal-string [:presto :Coercion/YYYYMMDDHHMMSSString->Temporal]
+  [driver _coercion_strategy expr]
+  (hsql/call :date_parse expr (hsql/raw "'%Y%m%d%H%i%s'")))
```

Comment:
We actually have an `hx/literal` function that will do the same thing but also put the single quotes in there for you. Not a big difference but a little nicer to look at

Reply:
It also escapes stuff to prevent SQL injections. Not important here since the strings aren't user-supplied, but it's another added benefit

Reply:
updated all the hsql/raw to hx/literal.